pkg/srtp/session.go

```diff
@@ -83,10 +83,10 @@ func (s *Session) WriteRTP(packet *rtp.Packet) (int, error) {
 		Header: rtp.Header{
 			Version:        2,
 			Marker:         packet.Marker,
-			PayloadType:    s.PayloadType,
+			PayloadType:    packet.PayloadType,
```

Comment:
This line and a few lines down is what was preventing me from changing the payload type and ssrc on the rtp packets